website/src/models/user.js

```diff
@@ -390,6 +390,15 @@ var UserSchema = new Schema({
       hatchPet: {type: Boolean, 'default': false},
       raisePet: {type: Boolean, 'default': false},
       streak: {type: Boolean, 'default': false}
+    },
+    improvementCategories: {
```

Comment:
@vIiRuS What route will be used to update these categories? I don't think we should tack these onto the [User PUT route's acceptable paths](https://github.com/HabitRPG/habitrpg/blob/c95950c679c018a30f07d6430b3e9fda61fac11e/website/src/controllers/api-v2/user.js#L285-L299), and instead should have it's own route to update the properties.